sdk/Lusid.Sdk.Tests/tutorials/Instruments/FxOptionExamples.cs

```diff
@@ -150,5 +152,110 @@ public void FxOptionPortfolioCashFlowsExample(ModelSelection.ModelEnum model, bo
             var fxOption = InstrumentExamples.CreateExampleFxOption(isDeliveryNotCash);
             CallLusidGetPortfolioCashFlowsEndpoint(fxOption, model);
         }
+
+        /// <summary>
+        /// Lifecycle management (i.e. conservation of PV on cashflow events) for fx option
```

Comment:
For these public docs, can we do "FX" and "PV" over its lowercase counterpart please? 
(I'm sure I've been guilty of otherwise)

Reply:
done